website_and_docs/content/documentation/webdriver/actions_api/keyboard.en.md

```diff
@@ -297,7 +297,18 @@ action.move_to_element(search).click().send_keys("send_keys", Keys.ENTER).perfor
 {{< /tab >}}
 {{< tab header="CSharp" >}}
 
-# Help us with a PR for code sample
+IWebDriver driver = new ChromeDriver();
+// Navigate to the url
+driver.Url = "https://www.selenium.dev";
+// Create an object of Action class
+Actions action = new Actions(driver);
+// Find google search box element
```

Comment:
As the above url navigates to selenium.dev, google search is not require in here, 
may be `search box element` should do